bin/pycbc_condition_hdf

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
```

Comment:
It seems weird to have a `pycbc_condition_strain` and a `pycbc_condition_hdf`, both doing very similar things. Can these not be combined?

Reply:
(Maybe these are sufficiently different as `condition_hdf` is writing a large data set ... Maybe the executable name is the problem ...)

Reply:
Ok, that makes sense. Maybe pycbc_data_store?

Reply:
Ok, I've changed the file name